datastore/bolt/api_key.go

```diff
@@ -2,54 +2,249 @@ package bolt
 
 import (
 	"context"
+	"encoding/json"
+	"math"
 
 	"github.com/frain-dev/convoy/datastore"
 	"go.etcd.io/bbolt"
 )
 
+const apiKeyBucketName = "apiKeys"
```

Comment:
Is there a reason why you didn't use the struct field in the apiKeyRepo?

Reply:
struct field is still being used. Just assigned a constant for the bucket name